components/swap/SurchargeCard.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="nft-card bg-k-grey-light !px-4 !py-6 flex flex-col items-center justify-between gap-4">
+    <div class="text-lg text-center font-bold capitalize">
+      {{ $t('swap.surchargeTitle') }}
+    </div>
+
+    <div class="flex flex-col gap-1 items-center justify-center rounded-2xl bg-white w-full h-full">
+      <div class="text-[31px]">
+        {{ formatted }}
+      </div>
+
+      <div class="text-k-grey">
+        ~ {{ usd }} $USD
+      </div>
+    </div>
+
+    <div class="text-xs text-center capitalize">
+      {{ $t(surcharge?.direction === 'Send' ? 'swap.surchargeIncludedWithYourItems' : 'swap.surchargeIncludedWithTheirItems') }}
```

Comment:
remove `?`  if it's not nullable 

```suggestion
      {{ $t(surcharge.direction === 'Send' ? 'swap.surchargeIncludedWithYourItems' : 'swap.surchargeIncludedWithTheirItems') }}
```

Reply:
updated